guides/rules/no-timers.md

```diff
@@ -0,0 +1,2 @@
+# No Timers
+
```

Comment:
I was hoping we could collaborate on what we should put in here. What do you think?